### PR TITLE
AEA-1977 Add validation endpoint to EPSAT

### PR DESCRIPTION
**Pull request by booshibooshi**

### CODE REVIEW

File: tool/site/client-react/src/components/textAreaField.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import {Textarea} from "nhsuk-react-components"
+import {Field} from "formik"
+
+export interface TextAreaFieldProps {
+  name: string
+  defaultValue?: string
+  style?: CSSProperties
+}
+
+const TextAreaField: FC<TextAreaFieldProps> = ({name, defaultValue, style}) => (
```
Comment (dan-jones-iw) [inline tool/site/client-react/src/components/textAreaField.tsx:11]: nhs has a Text Area component: https://nhsdigital.github.io/nhsuk-react-components/?path=/docs/components-textarea--standard

Comment (dan-jones-iw) [reply tool/site/client-react/src/components/textAreaField.tsx:11]: might be more visually appealing

File: tool/site/server_hapi/src/services/communication/sandbox-eps-client.ts [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import * as uuid from "uuid"
 import axios from "axios"
-import {Bundle, Claim, OperationOutcome, Parameters} from "fhir/r4"
+import {Bundle, Claim, FhirResource, OperationOutcome, Parameters} from "fhir/r4"
 import {asString, EpsClient, EpsResponse} from "./eps-client"
 
+
```
Comment (dan-jones-iw) [inline tool/site/server_hapi/src/services/communication/sandbox-eps-client.ts:6]: newline?

File: tool/site/client-react/src/components/textAreaField.tsx (hunk 1 above)
Comment (booshibooshi) [reply tool/site/client-react/src/components/textAreaField.tsx:11]: It is using the nhs component, this is a wrapper for formik to handle form submission

File: tool/site/client-react/src/pages/validatePage.tsx [hunk 3]
```diff
@@ -28,14 +32,17 @@ const ValidatePage: React.FC = () => {
...
               <Fieldset>
-                <TextAreaField
+                <Field
+                  id="validatePayload"
                   name="validatePayload"
-                  style={{height:"725px", width:"100%"}}/>
+                  as={Textarea}
+                  rows={20}
+                />
               </Fieldset>
```
Comment (dan-jones-iw) [inline tool/site/client-react/src/pages/validatePage.tsx:41]: not a huge deal, but could we style the FieldSet with margin-bottom instead?

**Review by dan-jones-iw — APPROVED** [review]

**Pull request merged**